install-extensions.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" -d foodies <<-EOSQL
+CREATE EXTENSION cube;
+CREATE EXTENSION earthdistance;
+EOSQL
```

Comment:
```
CREATE EXTENSION IF NOT EXISTS cube;
CREATE EXTENSION IF NOT EXISTS earthdistance;
```

I think I have an idea of how we can make this work without having to create a shell script. We can actually create a migration. Take a look at https://typeorm.io/#/migrations, and our repo also has some migrations too.

Basically, the migration would be to run the script above. I've configured migrations to run when our application starts. So we wouldn't need to worry about docker-compose and since we have the `IF NOT EXISTS`, we don't need to worry about installing it each time we start our application.